### PR TITLE
JSON load medical state machine 

### LINKED ISSUE #162: JSON to state machine
**Point of contact**: Mor Shemesh

We currently set our medical state machine in consts.py.
We would like to be able to dynamically and easily configure the state machine parameters (and perhaps in second step the whole state machine) from a JSON file.

Create and define a new JSON file with the relevant parameters, and enable importing it and creating a state machine from it.

### PR DESCRIPTION
**Pull request by benglewis**
Partially resolves #162 
Correct typos, make const classes dataclasses rather than NamedTuples so that they can be initialized with other values.
Migrate code from using from_file method to using from_json and modify and re-create the Python file as a new JSON file for loading the consts.

### CODE REVIEW

**Comment by benglewis** [comment]
Note:
This PR does not close #162 , an upcoming PR will implement the complete state machine in JSON.

File: README.md [hunk 1]
```diff
@@ -34,6 +34,23 @@
...
+## Creating a Constants JSON
+You can modify the constants for the models including the total execution steps, infection rates, medical state machine,
+detection test parameters and policy changes via a JSON file. An example can be found at 
+`tests/simulation_sanity_test/default_consts.json`.
+
+You can test your JSON file against the JSON schema for the
+constants, this file is found at `src/corona_hakab_model/consts-schema.json`.
+
+To do so, for those without PyCharm:
+1. Open `https://www.jsonschemavalidator.net` in your web browser.
```
Comment (bgmoshe) [inline README.md:47]: What you do if u are offline? (or god forbid in mamdas)

Comment (benglewis) [reply README.md:47]: PyCharm has a built in JSON schema validator:
https://www.jetbrains.com/help/idea/json.html#

**Comment by benglewis** [comment]
@bgmoshe @mor-m-shemesh @meirhalachmi 
אחד מכם יכול לאשר את זה?
@bgmoshe אתה רוצה שאני אסיף לריידמי את הקישור ששמתי כאן בהערה?

**Pull request merged**